frontend-v2/src/features/main/Sidebar.tsx

```diff
@@ -407,23 +533,26 @@ export default function Sidebar() {
         <Box
           component="nav"
           sx={{
-            width: { sm: drawerWidth },
+            width: {
+              sm: drawerExpandedWidth,
+            },
+            transition: "all .35s linear",
             flexShrink: { sm: 0 },
             height: "100vh",
-            backgroundColor: '#FBFBFA',
-            borderRight: '1px solid #DBD6D1'
           }}
           aria-label="mailbox folders"
-          id='simulations-portal'
+          id="simulations-portal"
         />
       )}
       <Box
         component="main"
+        id="main-content"
```

Comment:
I'd be inclined to set the expanded/collapsed className here, rather than modify the DOM directly.